contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -266,6 +266,9 @@ RETURNS INTEGER
 AS 'babelfishpg_tsql', 'checksum'
 LANGUAGE C IMMUTABLE PARALLEL SAFE;
 
+--- Dropping function sys.datetime2fromparts since there is a change in return type
+DROP FUNCTION IF EXISTS sys.datetime2fromparts(NUMERIC, NUMERIC, NUMERIC, NUMERIC, NUMERIC, NUMERIC, NUMERIC, NUMERIC) CASCADE;
```

Comment:
Seems redundant since below is the first time we are creating this function.

Reply:
removed